source/image-handler-custom-resource/setup.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 
 from setuptools import setup, find_packages
-from pip.req import parse_requirements
+from pip._internal.req import parse_requirements
```

Comment:
`pip` >= `10` does not expose `pip.req` and instead it was moved to `_internal`.